.github/workflows/test.yml

```diff
@@ -110,7 +113,7 @@ jobs:
         uses: "./.github/actions/check_clean"
 
       - name: Publish to NPM
-        if: env.IS_NOT_FORK == 'true' && env.IS_LATEST_NPM == 'true' && env.RELEASE == 'true'
+        if: env.IS_NOT_FORK == 'true' && env.IS_MRT_NODE == 'true' && env.RELEASE == 'true'
```

Comment:
Does the change still make sense here? All of the previous changes above are for pushing a bundle to MRT. But here it's a difference scenario: publishing to NPM.

Reply:
I decided to leave it as is. I prefer the more concrete name.